public/assets/open_distro_for_elasticsearch_logo_h.svg

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just curious how this file is generated.

Reply:
https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/master/public/assets/open_distro_for_elasticsearch_logo_h.svg